suites/squid/rgw/tier-4-rgw.yaml

```diff
@@ -248,3 +261,22 @@ tests:
       config:
         script-name: test_s3select.py
         config-file-name: test_s3select_query_gen_parquet_depth2.yaml
+
+  - test:
+      name: test s3select with depth1 queries on parquet objects
+      desc: test s3select with depth1 queries of incorrect syntax for checking rgw crashes on parquet objects
+      polarion-id: CEPH-83575176
+      module: sanity_rgw.py
+      config:
+        script-name: test_s3select.py
+        config-file-name: test_s3select_query_gen_parquet_depth1.yaml
+
+  # - test:
```

Comment:
@hmaheswa  is there any particular reason this testcase is commented?

Reply:
@anrao19 
you commented this test on reef in this PR actually https://github.com/red-hat-storage/cephci/pull/3796
it fails on squid also as it is targetted to 8.1
shall I uncomment this on squid?

Reply:
Since target is 8.1 squid testcase need to be there. lower versions like reef can be commented.
So please uncomment in squid

Reply:
done, pls review again